src/modules.js

```diff
@@ -327,9 +327,14 @@ function infermedicaModule() {
         user: {
           ...user,
           id: getUid(),
-          browser: browser.getBrowser(),
-          os: browser.getOS(),
-          platform: browser.getPlatform(),
+          browser: browser.browser,
+          os: browser.os,
+          platform: {
+            ...browser.device,
+            // Providing 'desktop' as default type value as ua-parser-js adds device.type only when explicitly defined in UA, e.g. 'Mobile Safari'
+            // More context - https://github.com/faisalman/ua-parser-js/issues/182#issuecomment-263115448
+            type: browser.device?.type ?? 'desktop',
+          },
```

Comment:
@kubajmarek Here I'd try to extract only the values we track.
`browser.device` returns `{ model: '', type: '', vendor: '' }`, but we track only type, so I'd skip passing all values by `...browser.device`.

Apart from that, all properties that we track should provide default values. 
My suggestion is to use `unknown` as a fallback value, but better confirm it with BI team.

Reply:
Created thread with BI, let's see what they say.
Personally, I'd say the more data the better. 😅